olp-cpp-sdk-core/src/http/curl/NetworkCurl.h

```diff
@@ -20,6 +20,7 @@
 #pragma once
 
 #include <olp/core/http/Network.h>
+#include <olp/core/http/NetworkRequest.h>
```

Comment:
is already included via Network.h

Reply:
We should prefer use principle of "if it's used it should be included". This allows to avoid to be dependent on side-effect includes.

Reply:
ok